src/stores/errors/errorsActions.ts

```diff
@@ -0,0 +1,27 @@
+import { Dispatch } from 'redux';
+
+export enum ErrorsActions {
+  RESET = '@error/reset',
+  SET = '@error/set',
+}
+
+const actionError = (error: unknown) => ({
```

Comment:
payload should be like "key": "value", where key is error name, and value is actual error